db/migrate/20170109221226_create_configuration_sources.rb

```diff
@@ -0,0 +1,17 @@
+class CreateConfigurationProjects < ActiveRecord::Migration[5.0]
+  def change
+    create_table :configuration_sources do |t|
+      t.belongs_to  :configuration_manager, :type => :bigint
+      t.string      :manager_ref
+      t.string      :name
+      t.string      :description
+      t.string      :scm_type
+      t.bigint      :scm_reference_id
+      t.boolean     :scm_clean
+      t.boolean     :scm_delete_on_update
+      t.boolean     :scm_update_on_launch
```

Comment:
For Ansible Inside, we'll probably be creating all projects with the same defaults.  I don't see the need to store that info on refresh.